.gitignore

```diff
@@ -66,3 +66,5 @@ fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots/**/*.png
 fastlane/test_output
+InventoryWatch/Assets.xcassets/.DS_Store
+InventoryWatch/Assets.xcassets/StatusBarLogo.imageset/.DS_Store
```

Comment:
I think you should just need to add a single `.DS_Store` entry to `.gitignore` - that should ignore all of these files throughout the whole directory. You'll also need to remove the `.DS_Store` files already added to the repo - see here: https://stackoverflow.com/questions/67224050/removing-ds-store-from-git-repo

Reply:
Yeah – I used the Github App for this, and it made multiple entries. Fixed.